### PR TITLE
Added composer support

### PR DESCRIPTION
**Pull request by johannes85**

### CODE REVIEW

**Comment by thekid** [comment]
Great! Merge at will

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "xp-framework/codegen",
+  "description": "XP Framework Code Generator Utility",
+  "type": "library",
+  "license": "BSD-3-Clause",
+  "require": {},
+  "require-dev": {
+      "xp-framework/core": "^6.0 | ^7.0 | ^8.0",
```
Comment (thekid) [inline composer.json]: What about 9.0?

Comment (johannes85) [reply composer.json]: Just wanted to merge this old branch. Then there is room for improvements.

Comment (thekid) [reply composer.json]: Sure, go ahead!

**Pull request merged**